roulette.py

```diff
@@ -1,92 +0,0 @@
-import random
-
-def rouletteNombre (betUser,money): #boucle pour répétition si user sort de l'intervalle [1,36]   
-    choiceUser=eval(input("Sur quel numéro voulez-vous poser votre jeton de 1 à 36 :"))
-    while choiceUser <1 or choiceUser >36: 
-        choiceUser=eval(input("Sur quel numéro voulez-vous poser votre jeton de 1 à 36 :"))
-    result=random.randrange(0,36)
-    print("La bille est tombée sur le", result)
-    if result==0:
-        print("la bille est tombée sur le 0, vous avez perdu")
-        return money
-    if choiceUser==result:
-            betUser=betUser*36
-            money=money+betUser
-            print("vous avez gagné",betUser,"€")
-            return money
-    else:
-            print("vous avez perdu", betUser, "€")
-            return money
-
-def rouletteStart(): #demande la mise
```

Comment:
This function should not be named rouletteStart if all it does is asking for the bet (as your little comment indicates).
If you rename it to `ask_bet_amount`, you can even remove your comment because it's going to be directly clear what it does